wsdl/ver10/schema/onvif.xsd

```diff
@@ -1807,12 +1807,106 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>Volume setting of the output. The applicable range is defined via the option AudioOutputOptions.OutputLevelRange.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="EQPresetToken" type="tt:ReferenceToken" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Optional audio EQ preset token to be applied to the audio output.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
 					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first Vendor then ONVIF -->
 				</xs:sequence>
 				<xs:anyAttribute processContents="lax"/>
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+
+	<xs:simpleType name="FrequencyRange">
+    <xs:restriction base="xs:string">
+        <xs:enumeration value="20Hz-60Hz">
+            <xs:annotation>
+                <xs:documentation>Sub-bass frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="60Hz-250Hz">
+            <xs:annotation>
+                <xs:documentation>Bass frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="250Hz-500Hz">
+            <xs:annotation>
+                <xs:documentation>Low midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="500Hz-2kHz">
+            <xs:annotation>
+                <xs:documentation>Midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="2kHz-4kHz">
+            <xs:annotation>
+                <xs:documentation>Upper midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="4kHz-20kHz">
+            <xs:annotation>
+                <xs:documentation>High frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+    </xs:restriction>
+</xs:simpleType>
+
+	<!--===============================-->
+	<!--          FrequencyGainPair            -->
+	<!--===============================-->
+	<xs:complexType name="FrequencyGainPair">
+    	<xs:sequence>
+        <xs:element name="FrequencyRange" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>Defined in tt:FrequencyRange, e.g., 20Hz-60Hz, 2kHz-4kHz.(Readonly)</xs:documentation>
```

Comment:
Is it necessasary to pre-define list or more dynamically let device return capabilites with ranges and gain.

Reply:
Device is free to choose its own, this is only a guideline.

Reply:
If the FrequencyRange enum is only for guideline, I suggest dropping the pre-defined from the spec to avoid confusion. Instead, add more documentation on what to expect if necessary. Also, a common guideline for EQ is hard, right? As it exists many different implementations for different products and use cases. For example, another guideline would be a 10-band EQ based on the octave band (as described here https://en.wikipedia.org/wiki/Octave_band, also coming from an IEC standard).

I also suggest moving from specifying a frequency range to instead set the center frequency of the band. This would increase possibility to compare between products as the range may differ depending on kind of filter and bandwidth.

Reply:
> If the FrequencyRange enum is only for guideline, I suggest dropping the pre-defined from the spec to avoid confusion. Instead, add more documentation on what to expect if necessary. Also, a common guideline for EQ is hard, right? As it exists many different implementations for different products and use cases. For example, another guideline would be a 10-band EQ based on the octave band (as described here https://en.wikipedia.org/wiki/Octave_band, also coming from an IEC standard).
> 
> I also suggest moving from specifying a frequency range to instead set the center frequency of the band. This would increase possibility to compare between products as the range may differ depending on kind of filter and bandwidth.

@robberos , agree with your suggestion will discuss this change in upcoming telco and will update it.

Reply:
@sujithhanwha @robberos I also found your suggestion interesting after reviewing your link.